ontology/yaml/resources/PLUMBING/entity_types/ABSTRACT.yaml

```diff
@@ -34,4 +34,35 @@ LVLM:
   - level_status
   opt_uses:
   - percentage_sensor
+
+SWCM:
+  guid: "66cfede5-b839-4504-82ac-60f89f7bbb45"
+  description: "Solenoid water valve control and monitoring."
+  is_abstract: true
+  uses:
+  - valve_command
+  - valve_status
   
+MWCM:
+  guid: "813c8a91-7f08-47ac-af20-95dda60f3211"
+  description: "Motorized water valve control and monitoring."
+  is_abstract: true
+  uses:
+  - valve_command
+  - valve_status
+
+WCR:
```

Comment:
Please move these to the GENERALTYPES.yaml file.

Reply:
WCR removed, was an artifact from original PR-794